node-backend/server.js

```diff
@@ -0,0 +1,19 @@
+var express = require("express");
```

Comment:
Let's use `const` and `let` for all variable declarations.

Reply:
Okay